components/performance_manager/service_worker_context_adapter.cc

```diff
@@ -77,12 +79,16 @@ void ServiceWorkerContextAdapter::RunningServiceWorker::RenderProcessExited(
 }
 
 void ServiceWorkerContextAdapter::RunningServiceWorker::
-    RenderProcessHostDestroyed(content::RenderProcessHost* host) {
-  // In --single-process mode, `RenderProcessExited` is never called as there is
-  // no render process. The worker nodes still need to be cleaned up before the
-  // process node.
+    InProcessRendererExiting(content::RenderProcessHost* host) {
```

Comment:
Should these changes gated by #ifdef COBALT ?

Reply:
He's cherry picking an upstream change, this seems correct